x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -99,7 +99,7 @@ export interface CurrentUpgrade {
   nbAgents: number;
   nbAgentsAck: number;
   version: string;
-  startTime: string;
+  startTime?: string;
```

Comment:
no start time means it runs immediately?

Reply:
Yes it run immediately without rolling window